UX-Guide-Metadata/techniques/onix.html

```diff
@@ -55,10 +55,10 @@ <h2>ONIX Metadata</h2>
 			</p>
 			
 			<p class="note">
-				ONIX messages describe products for the global book supply chain and will be sent from publisher or creator of the epub or digital audiobook to those who will make the products available for sale, lending or subscription.
+				ONIX messages describe products for the global book supply chain and will be sent from publisher or creator of the epub or digital full audio books to those who will make the products available for sale, lending or subscription.
```

Comment:
Isn't this instance actually referring to audiobooks since it talks of them separately from epub?

Reply:
@gregoriopellegrino left this for you.